libs/vkd3d-shader/dxbc.c

```diff
@@ -1538,30 +1538,19 @@ static bool shader_sm4_read_param(struct vkd3d_sm4_data *priv, const DWORD **ptr
         }
         m = *(*ptr)++;
 
-        switch (m)
+        switch (m & 0xc0)
         {
-            case 0x41:
-                *modifier = VKD3DSPSM_NEG;
-                break;
+            case 0x00: *modifier = VKD3DSPSM_NONE; break;
+            case 0x40: *modifier = VKD3DSPSM_NEG; break;
+            case 0x81: *modifier = VKD3DSPSM_ABS; break;
```

Comment:
Can 0x81 or 0xc1 occur here?

Reply:
Typo, this should be 0x80. 1 at the end can never occur because it's masked out.

Reply:
Ideally we'd have the flags defined somewhere, but the headers currently don't do that...